docs/Policy/Maintenance/Scenario_Maintenance_Policy.md

```diff
@@ -99,12 +104,14 @@ This will lead to two OCP clusters being provisioned in parallel (one will be th
 > Also we must ensure that a rehearsal job is run and passes based on the new expansion code.
 
 ## Scenario Deprecations
+
 > This is the responsibility of the PQE team.
 
 When a layered product version is no longer supported on the OpenShift release that it is being tested on we must remove the older version of the layered product from the scenario config file. Using the example above a scenario deprecation will just involve removing the test stanza for the older version that needs to be deprecated.
```

Comment:
```suggestion
When a layered product version is no longer supported on the OpenShift release that it is being tested on we must remove the older version of the layered product from the scenario config file. Using the example above, a scenario deprecation will just involve removing the test stanza for the older version that needs to be deprecated.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)